### DISCUSSION TITLE
Copy user preferences.

### DISCUSSION THREAD

**Discussion by msrt500** [netbox-community/netbox#6299] [Q&A] [answered]
Hi everyone!

How can i copy my preferences of interface to other users?

![user_prefer](https://user-images.githubusercontent.com/78883293/116328151-bd5c9500-a80b-11eb-8942-c2a62cda895a.png)

**Comment by ryanmerolle** [comment, marked as answer]
I think you could string together some API work to do that, but the easiest is to probably update the postgres table(s) themselves.

**Reply by msrt500** [reply]
I asked a question for a long time. I did the second option: I copied the settings from my account to the other users in postgre db. Thanks.
